client/branded/src/global-styles/tabs.scss

```
@@ -29,6 +25,66 @@
 }
 
 .tablist-wrapper {
-    background: var(--body-bg);
     border-bottom: 3px solid var(--border-color);
+    align-items: center;
+    &__full-width {
+        display: flex;
+        justify-content: space-around;
+        flex: 1;
+    }
+}
+
+.theme-redesign {
+    [data-reach-tabs] {
+        background: var(--body-bg);
+    }
+
+    [data-reach-tab-list] {
+        padding: 0;
+        &:first-child {
+            padding-left: 0;
+        }
+        &:last-child {
+            padding-right: 0;
+        }
+    }
+
+    [data-reach-tab] {
+        align-items: center;
+        letter-spacing: normal;
+        font-size: 0.75rem;
+        padding: 0 0.625rem;
+        margin: 0;
+        color: var(--body-color);
+        text-transform: none;
+        &:hover {
+            border-bottom: 3px solid var(--border-color);
+        }
+        &[data-selected] {
+            color: var(--body-color);
+            font-weight: bold;
+            border-bottom: 3px solid var(--brand-secondary);
+        }
+        &:focus-visible {
+            outline: none;
+            box-shadow: none;
+            .tablist-label {
```

Our style guide says to avoid descendant and child selectors, though I think this is a case where it makes sense. But, it's preferred to use a child selector if we don't actually need arbitrary-depth descendants.

```suggestion
            > .tablist-label {
```